codegen-server/python/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+
+plugins {
+    kotlin("jvm")
+    maven
+    `maven-publish`
+}
+
+description = "Generates Rust/Python server-side code from Smithy models"
+
+extra["displayName"] = "Smithy :: Rust :: Codegen :: Server :: Python"
+
+extra["moduleName"] = "software.amazon.smithy.rust.codegen.server.python"
+
+group = "software.amazon.smithy.rust.codegen.server.python.smithy"
+
+version = "0.1.0"
+
+val smithyVersion: String by project
+val kotestVersion: String by project
+
+dependencies {
+    implementation(project(":codegen"))
+    implementation(project(":codegen-server"))
+    implementation("software.amazon.smithy:smithy-aws-traits:$smithyVersion")
+    implementation("software.amazon.smithy:smithy-protocol-test-traits:$smithyVersion")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.6.1")
+    testImplementation("io.kotest:kotest-assertions-core-jvm:$kotestVersion")
+}
+
+// unlike the client-runtime, software.amazon.smithy.rust.codegen.smithy-kotlin codegen package is
+// not expected to run on Android...we can target 1.8
```

Comment:
This comment seems out of place. Did it originate from smithy-kotlin or something?